.github/actions/sync_criteria_check/action.yaml

```
@@ -32,6 +36,11 @@ runs:
       shell: bash
       run: |
         echo ::group::Generate job
+        return_zero_arg=""
+        if [[ "${{inputs.return_zero}}" != "false" ]]; then
```

Slightly more robust check:

```suggestion
        if ! ${{inputs.return_zero}}; then
```